src/App.js

```diff
@@ -1,16 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+
 
 const App = () => {
+  const [likes, setLikes] = useState(0);
```

Comment:
The number of liked messages should be determined from inspecting the current list of `chatMessages` data. Each chat entry _has_ a like attribute, which we should be using. So that also means we should be using state to store `chatMessages`, so that likes can be updated for each chat entry. 

```js
const [messages, setMessages] = useState(chatMessages)
```

We want to avoid introducing additional state when the existing data (`chatMessages`) provides the information we need (ie, each chat entry has a likes attribute), and without the risk of state getting out of sync. Remove `likes` state altogether.